.github/PULL_REQUEST_TEMPLATE/analyst_experience_team.md

```diff
@@ -0,0 +1,48 @@
+## Summary
+
+Summarize your PR. If it involves visual changes include a screenshot or gif.
+
+- **Related issue:** < LINK_TO_RELATED_ISSUE >
+- **Feature flag:** < FEATURE_FLAG_HERE >
+- **Docs ticket:** < LINK_TO_DOCS_TICKET_HERE >
+- **Schema changes:** < LINK_SCHEMA_CHANGE_TICKET_HERE >
+
+### How to test
+
+Provide steps for reviewers to know how to test your PR. Assume that a reviewer may have little to no context when determining how detailed this section should be.
+
+### Release Notes
+
+The text that appears in the Release Notes is pulled directly from your PR title, or a single paragraph of text that you specify in the PR description. See [Release Notes Guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process).
+
+### Definition of done
```

Comment:
I believe it would be great to have an extra point for our pipelines, to make sure the owner of the PR have considered to add new tests in the [Kibana QA QG](https://docs.elastic.dev/security-solution/teams/analyst-experience/automation/serverless/4.adding-tests.mdx) and [Elasticsearch Serverless Verify and Promote Pipeline](https://docs.elastic.dev/security-solution/teams/analyst-experience/automation/serverless/4.adding-tests.mdx)